tsdb/field_validator.go

```diff
@@ -42,6 +43,12 @@ func ValidateAndCreateFields(mf *MeasurementFields, point models.Point, skipSize
 		fieldKey := iter.FieldKey()
 		// Skip fields name "time", they are illegal.
 		if bytes.Equal(fieldKey, timeBytes) {
```

Comment:
Let's unify the two `timeBytes` declarations into one `tsdb.TimeBytes` in `shard.go`